vcpkg-root error with vcpkg 2020.04

When using master PMM on Ubuntu 18.04 with the newest version of vcpkg 2020.4, it fails with the following error:

```-- [pmm] Downloading vcpkg at 2020.04 ...
-- [download 100% complete]
-- [pmm] Extracting vcpkg archive...
-- [pmm] Bootstrapping the vcpkg tool (This may take a minute)...
-- [pmm] Testing bootstrapped vcpkg
-- [pmm] vcpkg successfully bootstrapped to /home/omaralv/.local/share/pmm/1.4.2/vcpkg-2020.04
-- [pmm] Using vcpkg executable: /home/omaralv/.local/share/pmm/1.4.2/vcpkg-2020.04/vcpkg
-- [pmm] Installing requirements with vcpkg

CMake Error at build/_pmm/1.4.2/vcpkg.cmake:185 (message):
  Failed to install requirements with vcpkg [1]:

  Error: Could not detect vcpkg-root.
```

--- a/pmm/conan.cmake
+++ b/pmm/conan.cmake
@@ -1,5 +1,5 @@
 set(PMM_CONAN_MIN_VERSION 1.21.0 CACHE INTERNAL "Minimum Conan version we support")
-set(PMM_CONAN_MAX_VERSION 1.24.99 CACHE INTERNAL "Maximum Conan version we support")
+set(PMM_CONAN_MAX_VERSION 1.25.99 CACHE INTERNAL "Maximum Conan version we support")
 
 option(PMM_CMAKE_MULTI "Use the cmake_multi generator for Conan" ON)
 

--- a/pmm/vcpkg.cmake
+++ b/pmm/vcpkg.cmake
@@ -80,8 +80,6 @@ function(_pmm_ensure_vcpkg dir rev)
     file(REMOVE_RECURSE "${dir}")
     file(RENAME "${vcpkg_root}" "${dir}")
     _pmm_log("vcpkg successfully bootstrapped to ${dir}")
-    # Fix for "Could not detect vcpkg-root."
-    execute_process(COMMAND ${CMAKE_COMMAND} -E sleep 1)
 endfunction()
 
 function(_pmm_vcpkg_default_triplet out)

--- a/pmm/vcpkg.cmake
+++ b/pmm/vcpkg.cmake
@@ -80,6 +80,8 @@ function(_pmm_ensure_vcpkg dir rev)
     file(REMOVE_RECURSE "${dir}")
     file(RENAME "${vcpkg_root}" "${dir}")
     _pmm_log("vcpkg successfully bootstrapped to ${dir}")
+    # Fix for "Could not detect vcpkg-root."
+    execute_process(COMMAND ${CMAKE_COMMAND} -E sleep 1)
 endfunction()
 
 function(_pmm_vcpkg_default_triplet out)
@@ -200,13 +202,7 @@ function(_pmm_vcpkg)
     )
 
     if (NOT DEFINED ARG_REVISION)
-        # This is just a random revision people can plop down in for the REVISION
-        # argument. There isn't anything significant about this particular
-        # revision, other than being the revision of the `master` branch at the
-        # time I typed this comment. If you are modifying PMM, feel free to
-        # change this revision number to whatever is the latest in the vcpkg
-        # repository. (https://github.com/Microsoft/vcpkg)
-        message(FATAL_ERROR "Using pmm(VCPKG) requires a REVISION argument. Try `REVISION 2020.01`")
+        message(FATAL_ERROR "Using pmm(VCPKG) requires a REVISION argument. Try `REVISION 2020.04`")
     endif ()
     if (NOT DEFINED ARG_TRIPLET)
         _pmm_vcpkg_default_triplet(ARG_TRIPLET)
@@ -226,6 +222,7 @@ function(_pmm_vcpkg)
         _pmm_generate_vcpkg_cli_scripts()
         _pmm_log("Installing requirements with vcpkg")
         set(cmd ${CMAKE_COMMAND} -E env
+                VCPKG_ROOT=${vcpkg_inst_dir}
                 CC=${CMAKE_C_COMPILER}
                 CXX=${CMAKE_CXX_COMPILER}
                 "${PMM_VCPKG_EXECUTABLE}" install

--- a/tests/vcpkg/CMakeLists.txt
+++ b/tests/vcpkg/CMakeLists.txt
@@ -8,7 +8,7 @@ set(PMM_DEBUG TRUE)
 include(../../pmm.cmake)
 
 pmm(VCPKG
-    REVISION 2020.04
+    REVISION e792d1c171a18d32401db0f95251515cc15056f5
     REQUIRES catch2
     DEBUG
     )

--- a/tests/vcpkg/CMakeLists.txt
+++ b/tests/vcpkg/CMakeLists.txt
@@ -8,7 +8,7 @@ set(PMM_DEBUG TRUE)
 include(../../pmm.cmake)
 
 pmm(VCPKG
-    REVISION 2020.01
+    REVISION 2020.04
     REQUIRES catch2
     DEBUG
     )

